conans/client/cmd/new.py

```diff
@@ -22,10 +22,14 @@ class {package_name}Conan(ConanFile):
     description = "<Description of {package_name} here>"
     topics = ("<Put some tag here>", "<here>", "<and here>")
     settings = "os", "compiler", "build_type", "arch"
-    options = {{"shared": [True, False]}}
-    default_options = {{"shared": False}}
+    options = {{"shared": [True, False], "fPIC": [True, False]}}
+    default_options = {{"shared": False, "fPIC": True}}
     generators = "cmake"
 
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
```

Comment:
I am fairly certain this is also expected during review

```suggestion
    def config_options(self):
        if self.settings.os == "Windows":
            del self.options.fPIC

    def configure(self):
        if self.options.shared:
            del self.options.fPIC
```

Reply:
Not completely sure, many recipes do not implement it yet. I would say wait until this is discussed a bit more, seems that some Conan built-in helpers would be better.

Reply:
agree
```
    def config_options(self):
        if self.settings.os == "Windows":
            del self.options.fPIC
```
this code has 361 occurrences on CCI
```
        if self.options.shared:
            del self.options.fPIC
```
this one is only 264.
so, probably, it might be too early to enforce that, as it's not an established practice on CCI.
earlier we also had proposal for so called `shared_library_package_id` to be used in recipes, I don't remember what was a conclusion.
we need to talk about these two IMO before we start to recommend them.